scripts/apps/monitoring/styles/monitoring.scss

```diff
@@ -287,18 +287,15 @@
     }
 }
 
-.swimlane2 {
-    display: flex;
-    justify-content: space-between;
-
-    > * {
-        flex-basis: 0;
-        flex-grow: 1;
-        min-width: 250px;
-    }
+.sd-kanban-list {
+    position: absolute;
```

Comment:
Why do you need to use absolute positioning? The previous implementation I've done worked fine without it. I think scrollable contents and static labels can be done with flexbox, can't it?

Reply:
seems like it can